gpkit/tests/t_sub.py

```diff
@@ -354,7 +364,7 @@ def test_vector_sub(self):
             # issue1077 links to a case that failed for SPs only
             m = Model(x.prod(), [x + y >= 1, y <= ymax])
 
-        m.substitutions.update({"ymax": [0.3, 0.5, 0.8]})
+        m.substitutions["ymax"] = [0.3, 0.5, 0.8]
```

Comment:
Because I realized we weren't testing square-bracket substitution changing anywhere